dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/TemplateLayoutRow.java

```diff
@@ -17,9 +20,17 @@
     private String value;
     private String id;
 
-    public int columnsCount;
-    public Integer[] gridWidths;
-    public List<TemplateLayoutColumn> columns;
+    private Integer[] gridWidths;
+    private List<TemplateLayoutColumn> columns;
+
+    @JsonCreator
+    public TemplateLayoutRow(@JsonProperty("columns") List<TemplateLayoutColumn> columns) {
+        this.columns = columns;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "columns". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)